pkg/lib/page.scss

```diff
@@ -549,3 +549,11 @@ input[type=number]:not(.pf-c-form-control) {
     line-height: 1.5;
     border-radius: .2rem;
 }
+
+html:not(.index_page) body {
+    // Ensure UI fills the entire page (and does not run over)
+    > div:first-child {
```

Comment:
This is wrong - will not affected pages which have other div before the main page div, which is very common. Removing it to see what happens.